bigbone-rx/src/main/kotlin/social/bigbone/rx/admin/RxAdminTrendMethods.kt

```diff
@@ -0,0 +1,48 @@
+package social.bigbone.rx.admin
+
+import io.reactivex.rxjava3.core.Single
+import social.bigbone.MastodonClient
+import social.bigbone.MastodonRequest
+import social.bigbone.api.entity.Status
+import social.bigbone.api.entity.TrendsLink
+import social.bigbone.api.entity.admin.AdminTag
+import social.bigbone.api.method.admin.AdminTrendMethods
+
+/**
+ * Reactive implementation of [AdminTrendMethods].
+ *
+ * Obtain data about trending links, statuses, and tags on the administered server.
+ *
+ * @see <a href="https://docs.joinmastodon.org/methods/admin/trends/">Mastodon admin/trends API methods</a>
+ */
+class RxAdminTrendMethods(client: MastodonClient) {
+
+    private val adminTrendMethods = AdminTrendMethods(client)
+
+    /**
+     * Links that have been shared more than others, including unapproved and unreviewed links.
+     *
+     * @see <a href="https://docs.joinmastodon.org/methods/admin/trends/#links">Mastodon API documentation: admin/trends/#links</a>
+     */
+    fun getTrendingLinks(): Single<MastodonRequest<List<TrendsLink>>> = Single.fromCallable {
+        adminTrendMethods.getTrendingLinks()
```

Comment:
I think the `.execute()` call is missing here (and in the other methods below).

Reply:
Whoops! Fixed. 😊